packages/network-controller/tests/NetworkController.test.ts

```diff
@@ -939,139 +972,749 @@ describe('NetworkController', () => {
       });
     });
 
-    describe('if a provider has been set', () => {
```

Comment:
This block seemed unnecessary; we can consider the case where the provider is set as the default

Reply:
I can get behind this — in that case what do you think about taking the "does not change network in state" test above out of its `describe`? So it would get renamed to something like "does not change network in state if a provider has not been set".